x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/feature_table/feature_table_expanded_row.tsx

```diff
@@ -110,6 +112,8 @@ export const FeatureTableExpandedRow = ({
         </div>
       </EuiFlexItem>
       {feature.getSubFeatures().map((subFeature) => {
+        const disabledDueToSpaceSelection = subFeature.requireAllSpaces && !allSpacesSelected;
```

Comment:
I believe it is good practice to name boolean vars to start with a `is`, `are`, `does`, `has`, `have`, or some such verb that answers to a `yes` or a `no` answer. It is quicker to grasp what the variable is for if no types are attached to it when skimming through the code. 

So maybe rename this to `isDisabledForSpaceSelectionReason` or something on those lines.